src/utils/libfabric/libfabric_topology.cpp

```
@@ -195,6 +195,48 @@ nixlLibfabricTopology::isValidDevice(const std::string &efa_device) const {
     return std::find(all_devices.begin(), all_devices.end(), efa_device) != all_devices.end();
 }
 
+nixl_status_t
+nixlLibfabricTopology::hasPCIeDevices(bool &hasDevice) {
```

Cppcheck (2.20.0)

[style] The function 'hasPCIeDevices' is never used.

(unusedFunction)